### PR TITLE
Remove orchestrator from rook task and suite

### PR DESCRIPTION
**Pull request by josephsawaya**
This PR removes orchestrator commands from the
Rook task and the Rook test suite because the Rook
orchestrator is not being maintained, and the Rook
orchestrator CLI is obsolete. This should also
clarify the issue:

https://tracker.ceph.com/issues/53680

Signed-off-by: Joseph Sawaya <jsawaya@redhat.com>

## Checklist
- Tracker (select at least one)
  - [X] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [X] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [X] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [X] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Review by adk3798 — APPROVED** [review]

**Comment by ljflores** [comment]
jenkins test api

**Review by ljflores — APPROVED** [review]

**Comment by ljflores** [comment]
Thanks for this @josephsawaya !

**Comment by ljflores** [comment]
@travisn this PR passed the teuthology tests, so I think it is ready to merge. Tagging you though to make sure this is okay with rook.

**Comment by ljflores** [comment]
jenkins test api

File: qa/suites/orch/rook/smoke/1-rook.yaml [hunk 1]
```diff
@@ -1,8 +1,2 @@
 tasks:
```
Comment (jeffvance) [inline qa/suites/orch/rook/smoke/1-rook.yaml:1]: do you still need this file? Is pretty much empty now...

File: qa/tasks/rook.py [hunk 2]
```diff
@@ -367,6 +367,30 @@ def rook_cluster(ctx, config):
...
+                        'volumeClaimTemplates': [
+                            {
+                                'metadata': {'name': 'data'},
+                                'spec': {
+                                    'resources': {
+                                        'requests': {
+                                            'storage': '10Gi'  # <= (lte) the actual PV size
+                                        }
+                                    },
+                                    'storageClassName': 'scratch',
```
Comment (travisn) [inline qa/tasks/rook.py:385]: Are the PVs from a scratch storage class defined somewhere in the test already? If not, we won't be seeing OSDs created.

Comment (josephsawaya) [reply qa/tasks/rook.py:385]: https://github.com/ceph/ceph/blob/57c2ae079b42cde0e8f3deedc760183af81503b2/qa/tasks/kubeadm.py#L452

**Comment by ljflores** [comment]
@travisn @adk3798 etc. can we merge this?

**Pull request merged**